src/components/CountDownTimer.tsx

```diff
@@ -0,0 +1,108 @@
+import { CountdownCircleTimer } from "react-countdown-circle-timer";
+import "../styles.css";
+
+const minuteSeconds = 60;
+const hourSeconds = 3600;
+const daySeconds = 86400;
+
+const timerProps = {
+  isPlaying: true,
+  size: 120,
+  strokeWidth: 6
+};
+
+const renderTime = (dimension: any, time: number) => {
+  return (
+    <div className="time-wrapper">
+      <div className="time">{time}</div>
+      <div>{dimension}</div>
+    </div>
+  );
+};
+
+const getTimeSeconds = (time: number) => (minuteSeconds - time) | 0;
+const getTimeMinutes = (time: number) =>
+  ((time % hourSeconds) / minuteSeconds) | 0;
+const getTimeHours = (time: number) => ((time % daySeconds) / hourSeconds) | 0;
+const getTimeDays = (time: number) => (time / daySeconds) | 0;
+
+export default function CountDownTimer() {
+  const stratTime = Date.now() / 1000; // use UNIX timestamp in seconds
+  const endTime = stratTime + 243248; // use UNIX timestamp in seconds
+
+  const remainingTime = endTime - stratTime;
+  const days = Math.ceil(remainingTime / daySeconds);
+  const daysDuration = days * daySeconds;
+
+  return (
+    <div className="App flex">
+      <CountdownCircleTimer
+        {...timerProps}
+        colors="#FAEE6C"
+        duration={daysDuration}
+        initialRemainingTime={remainingTime}
+        size={225}
```

Comment:
seems like you are overwriteing the size prop from timerProps. Might be worth double checking which props you can move to the timerProps variable

Reply:
I have to define it for each of the circles cause there is 4 of them :)

Reply:
yeah, but it is defined as 225 for each of them. So why not make the size property of timerProps 225?

Reply:
Yep I get you, just changed that also